awx/plugins/inventory/foreman.py

```diff
@@ -276,6 +360,91 @@ def to_safe(self, word):
 
     def update_cache(self, scan_only_new_hosts=False):
         """Make calls to foreman and save the output in a cache"""
+        use_inventory_report = self._use_inventory_report()
+        if use_inventory_report:
+            self._update_cache_inventory(scan_only_new_hosts)
+        else:
+            self._update_cache_host_api(scan_only_new_hosts)
+
+    def _update_cache_inventory(self, scan_only_new_hosts):
+        self.groups = dict()
+        self.hosts = dict()
+        inventory_report_response = self._post_request()
+        host_data = json.loads(inventory_report_response)
+        for host in host_data:
+            if not(host) or (host["Name"] in self.cache.keys() and scan_only_new_hosts):
```

Comment:
all keys are lower cased, so e.g. `Name` need to be `name`, `Content Attributes` need to be `content_attributes`